### PR TITLE
In progress updates to App parameter styling TASK-701

### PR DESCRIPTION
**Pull request by msneddon**
Includes:
 - mouseover highlight of rows
 - removes most borders around parameter elements
 - changes the style of the feedback indicator to a line instead of an arrow
 - moves the 'more info' button per parameter to a click on the parameter field name
 - remove numbering in list inputs

### CODE REVIEW

**Comment by msneddon** [comment]
This PR is roughly tested on a handful of Apps (which seem to use the micro and compact variants), but I'm not sure the context or exactly how to test for all the variants of the parameter field widgets, or when the 'View' set of widgets are used instead of the 'Input'.

File: kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetFull.js [hunk 1]
```diff
@@ -316,15 +323,27 @@ define([
...
                     ]),
-                    div({ class: 'input-group col-md-9', style: { xwidth: '100%' } }, [
+                    div({ class: 'input-group col-md-10', style: { xwidth: '100%' } }, [
                         div({ dataElement: 'input-control' }),
-                        div({ class: 'input-group-addon', dataElement: 'feedback', style: { width: '30px', padding: '0' } }, [
+                        div({ class: 'input-group-addon kb-input-group-addon kb-app-field-feedback', dataElement: 'feedback', style: { width: '30px', padding: '0' } }, [
                             div({ dataElement: 'indicator' })
-                        ]),
-                        div({ class: 'input-group-addon', style: { width: '30px', padding: '0' } }, [
+                        ])/*,
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetFull.js]: Is this going to get uncommented? Or can it just be deleted?

File: kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidget.js [hunk 2]
```diff
@@ -347,10 +347,10 @@ define([
                     ]),
                     div({ class: 'input-group col-md-9', style: { xwidth: '100%' } }, [
                         div({ dataElement: 'input-control' }),
-                        div({ class: 'input-group-addon', dataElement: 'feedback', style: { width: '30px', padding: '0' } }, [
+                        div({ class: 'input-group-addon kb-input-group-addon kb-app-field-feedback', dataElement: 'feedback', style: { width: '30px', padding: '0' } }, [
                             div({ dataElement: 'indicator' })
-                        ]),
-                        div({ class: 'input-group-addon', style: { width: '30px', padding: '0' } }, [
+                        ]), /*
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidget.js]: Can this just be deleted?

File: kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetCompact.js [hunk 3]
```diff
@@ -344,20 +365,24 @@ define([
...
-                                dataElement: 'indicator'
+                                dataElement: 'indicator',
+                                style: {
+                                    width: '3px'
+                                }
                             })
                         ]),
-                        div({
-                            class: 'input-group-addon',
+                        /*div({
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetCompact.js]: Can this just be deleted?

Comment (msneddon) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetCompact.js]: Yes- all these blocks should be removed if we are going to stick with removing the current 'info' button.  I probably can't get to it though until much later this afternoon or tomorrow morning though...

Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetCompact.js]: Ok. Let's put it on ci now and revisit as needed.

**Pull request merged**